Projekt 7.py

```diff
@@ -21,6 +34,12 @@ def __repr__(self):
         return f'Expense(id={self.id!r}, description={self.description!r}, amount={self.amount!r})'
```

Comment:
Zastanawiam się czy jeśli robimy __repr__ to może warto też przygotować metodę __str__?